doc/doxygen/references.bib

```diff
@@ -122,7 +122,6 @@ @phdthesis{Bec95
   author =  {R. Becker},
   title =   {An Adaptive Finite Element Method for the Incompressible {N}avier--{S}tokes Equations on Time-dependent Domains},
   school =  {Universit{\"a}t Heidelberg},
-  type =    {Dissertation},
   year =    1995
```

Comment:
While there, would you add a link to the thesis as well?
Either https://katalog.ub.uni-heidelberg.de/titel/9608091 or https://www.researchgate.net/publication/2818080